gdal_subsetter/transform.py

```diff
@@ -1012,301 +933,3 @@ def _mask_band(self, tmp_ds: gdal.Dataset, band_index: int,
         out_mskband.WriteArray(out_msk)
         out_mskband.FlushCache()
         out_band.FlushCache()
-
```

Comment:
All this below is migrated to `reformat.py`. See comment on that module describing the minor structural changes - no functional changes made.